
this fixes:

pkgmogrify: File openal.p5m line 59: Malformed action at position: 48: missing value:
    file path=usr/share/openal/hrtf/Default HRTF.mhr

--- openal-soft-1.23.1/CMakeLists.txt.orig	2025-05-26 20:56:12.771585014 +0300
+++ openal-soft-1.23.1/CMakeLists.txt	2025-05-26 20:56:51.032486313 +0300
@@ -1243,7 +1243,7 @@
         set(ALC_OBJS  ${ALC_OBJS} "${outfile}")
     endmacro()
 
-    make_hrtf_header("Default HRTF.mhr" "default_hrtf")
+    make_hrtf_header("Default-HRTF.mhr" "default_hrtf")
 endif()
 
 
